test/common/network/matching/inputs_integration_test.cc

```diff
@@ -72,7 +101,8 @@ TEST_F(InputsIntegrationTest, DestinationPortInput) {
   initialize("DestinationPortInput", "8080");
 
   Network::MockConnectionSocket socket;
-  MatchingDataImpl data(socket);
+  StreamInfo::FilterStateImpl filter_state(StreamInfo::FilterState::LifeSpan::FilterChain);
```

Comment:
Same here and elsewhere, please change to Connection to make it more realistic.

Reply:
done